catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -0,0 +1,376 @@
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>npm test</id>
+                        <goals>
+                            <goal>npm</goal>
+                        </goals>
+                        <phase>test</phase>
+                        <configuration>
+                            <skip>${skipTests}</skip>
+                            <arguments>test</arguments>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <instructions>
+                        <Embed-Dependency>
+                            boon,
+                            camel-http-common,
+                            camel-http,
+                            camel-core-osgi,
+                            httpclient,
+                            httpcore,
+                            platform-error-servlet,
+                            proxy-camel-servlet,
+                            proxy-camel-route,
+                            catalog-core-api-impl;scope=!test,
+                            platform-util,
+                            platform-util-unavailableurls,
+                            versioning-common,
+                            spark-core,
+                            gt-cql,
+                            spatial4j,
+                            commons-lang3,
+                            quartz,
+                            quartz-jobs,
+                            c3p0,
+                            terracotta-toolkit-runtime-ee,
+                            ehcache-terracotta,
+                            ehcache,
+                            ddf-security-common
+                        </Embed-Dependency>
+                        <Web-ContextPath>/search/catalog</Web-ContextPath>
+                        <_wab>src/main/webapp,${project.build.directory}/webapp</_wab>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Import-Package>
+                            org.eclipse.jetty.server,
+                            org.eclipse.jetty.server.handler,
+                            org.eclipse.jetty.servlets,
+                            javax.servlet.http;version="[3,4)",
+                            javax.servlet;version="[3,4)",
+                            org.apache.commons.logging; version="[1.1.0,2.0.0)",
+                            !sun.reflect,
+                            !com.sun.management,
+                            !org.eclipse.jetty.websocket.*,
+                            !org.noggit,
+                            !commonj.work.*,
+                            !oracle.sql.*,
```

Comment:
Can we add dependency exclusions to remove these imports instead of an explicit ignore in the import?